OsmAnd.xcodeproj/project.pbxproj

```diff
@@ -666,6 +666,7 @@
 		466A01802BFBA9B7008CC50E /* QuickActionType.swift in Sources */ = {isa = PBXBuildFile; fileRef = 466A017F2BFBA9B7008CC50E /* QuickActionType.swift */; };
```

Comment:
1) The pin on the map doesn't move along the segmented track.
2) The app freezes significantly when moving the finger across the graph.
In the master branch, it works fine.

Reply:
[EuroVelo.6.-.developed.2.gpx.zip](https://github.com/user-attachments/files/17730026/EuroVelo.6.-.developed.2.gpx.zip)